src/main/java/org/jenkinsci/plugins/pagerduty/changeevents/ChangeEvents.java

```diff
@@ -74,7 +78,14 @@ public void perform(Run<?, ?> build, @Nonnull FilePath workspace, @Nonnull Launc
         if ((result == Result.SUCCESS && createOnSuccess) || (result == Result.FAILURE && createOnFailure)
                 || (result == Result.UNSTABLE && createOnUnstable) || (result == Result.ABORTED && createOnAborted)
                 || (result == Result.NOT_BUILT && createOnNotBuilt)) {
-            new ChangeEventSender().send(integrationKey, build, listener);
+        	if(summaryText == null || summaryText.equals(""))
+        	{
+        		new ChangeEventSender().send(integrationKey, build, listener);
+        	}
+    		else
+        	{
+        		new ChangeEventSender().send(integrationKey, summaryText, build, listener);
```

Comment:
The suggestion above should boil this if block down to this one line, as the summaryText will be null and use the old behavior.